Documentation/contributing/testing/e2e.rst

```diff
@@ -1,5 +1,5 @@
 .. only:: not (epub or latex or html)
-  
+
```

Comment:
Please write the example in the commit message, slack links will expire almost within a month

Reply:
Done